lib/ripper-tags.rb

```diff
@@ -53,6 +55,27 @@ def self.option_parser(options)
           options.exclude << pattern
         end
       end
+      opts.on("--excmd=number", "Use EX number command to locate tags.") do |excmd|
+        if excmd == "number"
+          options.excmd = excmd
+        else
+          $stderr.puts "Error: excmd %p is not supported" % excmd
```

Comment:
Let's just save any value passed to `options.excmd` and not raise an error on any unsupported value.